<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenWritingClass</class>
 <widget class="QMainWindow" name="ScreenWritingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScreenWriting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="descriptionLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="actLabel">
            <property name="text">
             <string>Act</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="actLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="reviseButton">
        <property name="text">
         <string>Revise</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="developButton">
        <property name="text">
         <string>Develop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="savePlotButton">
        <property name="text">
         <string>Save plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ScreenWriting.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>reviseButton</sender>
   <signal>clicked()</signal>
   <receiver>ScreenWritingClass</receiver>
   <slot>reviseIdea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>developButton</sender>
   <signal>clicked()</signal>
   <receiver>ScreenWritingClass</receiver>
   <slot>developIdea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savePlotButton</sender>
   <signal>clicked()</signal>
   <receiver>ScreenWritingClass</receiver>
   <slot>savePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>ScreenWritingClass</receiver>
   <slot>addIdea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reviseIdea()</slot>
  <slot>developIdea()</slot>
  <slot>savePlot()</slot>
  <slot>addIdea()</slot>
 </slots>
</ui>
